recirq/kpz/kpz.ipynb

```diff
@@ -0,0 +1,746 @@
+      "text/plain": [
+       "<Figure size 1200x800 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "res_amplitudes.plot_histogram()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e0b743fc",
+   "metadata": {},
+   "source": [
+    "Finally, we are interested in the mean, variance, skewness, and kurtosis of this distribution. These are stored in `res_amplitudes.mean`, `res_amplitudes.variance`, `res_amplitudes.skewness`, and `res_amplitudes.kurtosis`, and similarly for `res_bitstrs`.\n",
+    "\n",
+    "These quantities all have statistical uncertainty because we are sampling initial states. The statistical uncertainties can be computed with the `jackknife_mean()`, `jackknife_variance()`, `jackknife_skew()`, and `jackknife_kurtosis()` methods of either `res_amplitudes` or `res_bitstrs`. In particular, we have"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 45,
+   "id": "dac5a56a",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "mean = 1.55 ± 0.05\n",
+      "variance = 2.9 ± 0.1\n",
+      "skewness = -0.33 ± 0.05\n",
+      "kurtosis = -0.3 ± 0.1\n"
+     ]
+    }
+   ],
+   "source": [
+    "print('mean = {:.2f} ± {:.2f}'.format(res_amplitudes.mean, res_amplitudes.jackknife_mean()))\n",
```

Comment:
Using f-strings is more readable:
`print(f'mean = {(res_amplitudes.mean:.2f} ± {res_amplitudes.jackknife_mean()):.2f}')`